src/entities/DealTokenSwap.ts

```diff
@@ -84,6 +84,7 @@ export class DealTokenSwap implements IDeal {
       this.registrationData = await this.dataSourceDeals.get<IDealRegistrationTokenSwap>(this.rootData.registration);
       const discussionsMap = await this.dataSourceDeals.get<Record<string, string> | undefined>(this.rootData.discussions);
       this.clauseDiscussions = new Map(Object.entries(discussionsMap ?? {}));
+      this.status = this.rootData.status;
```

Comment:
fake "status" should not be coming from Ceramic.  I think we need this coming from elsewhere for Deal entity stuff.

Reply:
Ideally we are deriving status by whatever logic it is supposed to be derived from

Reply:
How about for now we just have each Deal return a random value for status.

Reply:
I've removed the ceramic implementation and added a (temporary) random status distributer in the DealTokenSwap entity.